web/src/containers/TopBtns.js

```diff
@@ -0,0 +1,53 @@
+import PropTypes from 'prop-types';
+import React, { Component, Fragment } from 'react';
+import { Link } from 'react-router-dom';
+import { connect } from 'react-redux';
+
+import { t } from '../i18n';
+import { logout } from '../actions/auth';
+
+class TopBtns extends Component {
+  handleLogout = e => {
+    e.preventDefault();
+    this.props.logout();
+  };
+
+  render() {
+    const { authenticated } = this.props;
+
+    return (
+      <header className="clearfix mb2">
+        <div className="right h5">
+          {authenticated ? (
+            <Fragment>
+              <button type="button" className="btn btn-primary px1 py-tiny mr1">
+                {t('dashboard')}
+              </button>
```

Comment:
The "state dashboard" button is a placeholder I guess?

Reply:
yes!